--- sysstat-12.5.4/Makefile.in~	2021-02-14 17:59:21.000000000 +0100
+++ sysstat-12.5.4/Makefile.in	2021-06-28 13:45:06.130150947 +0200
@@ -599,7 +599,8 @@ ifneq ($(IGNORE_FILE_ATTRIBUTES),y)
 endif
 ifdef SYSTEMD_UNIT_DIR
 	mkdir -p $(DESTDIR)$(SYSTEMD_UNIT_DIR)
-else
+endif
+ifeq ($(INSTALL_CRON),y)
 	if [ -d /etc/cron.d ]; then \
 		mkdir -p $(DESTDIR)/etc/cron.d; \
 	elif [ -d /etc/cron.hourly -a -d /etc/cron.daily ]; then \
