diff -burN sysstat-5.0.3.orig/Makefile sysstat-5.0.3/Makefile
--- sysstat-5.0.3.orig/Makefile	2004-04-09 22:37:10.959886336 +0200
+++ sysstat-5.0.3/Makefile	2004-04-09 23:02:49.954923528 +0200
@@ -116,7 +116,7 @@
 	$(CC) -o $@ $(CFLAGS) $(DFLAGS) $< $(LFLAGS)
 
 ifdef REQUIRE_NLS
-locales: nls/fr/$(PACKAGE).mo nls/de/$(PACKAGE).mo nls/es/$(PACKAGE).mo nls/pt/$(PACKAGE).mo nls/af/$(PACKAGE).mo nls/nb_NO/$(PACKAGE).mo nls/nn_NO/$(PACKAGE).mo nls/it/$(PACKAGE).mo nls/ru/$(PACKAGE).mo nls/ro/$(PACKAGE).mo nls/pl/$(PACKAGE).mo nls/sk/$(PACKAGE).mo
+locales: nls/fr/$(PACKAGE).mo nls/de/$(PACKAGE).mo nls/es/$(PACKAGE).mo nls/pt/$(PACKAGE).mo nls/af/$(PACKAGE).mo nls/nb/$(PACKAGE).mo nls/nn/$(PACKAGE).mo nls/it/$(PACKAGE).mo nls/ru/$(PACKAGE).mo nls/ro/$(PACKAGE).mo nls/pl/$(PACKAGE).mo nls/sk/$(PACKAGE).mo
 else
 locales:
 endif
@@ -136,11 +136,11 @@
 nls/af/$(PACKAGE).mo: nls/af/$(PACKAGE).po
 	$(MSGFMT) -o nls/af/$(PACKAGE).mo nls/af/$(PACKAGE).po
 
-nls/nb_NO/$(PACKAGE).mo: nls/nb_NO/$(PACKAGE).po
-	$(MSGFMT) -o nls/nb_NO/$(PACKAGE).mo nls/nb_NO/$(PACKAGE).po
+nls/nb/$(PACKAGE).mo: nls/nb/$(PACKAGE).po
+	$(MSGFMT) -o nls/nb/$(PACKAGE).mo nls/nb/$(PACKAGE).po
 
-nls/nn_NO/$(PACKAGE).mo: nls/nn_NO/$(PACKAGE).po
-	$(MSGFMT) -o nls/nn_NO/$(PACKAGE).mo nls/nn_NO/$(PACKAGE).po
+nls/nn/$(PACKAGE).mo: nls/nn/$(PACKAGE).po
+	$(MSGFMT) -o nls/nn/$(PACKAGE).mo nls/nn/$(PACKAGE).po
 
 nls/it/$(PACKAGE).mo: nls/it/$(PACKAGE).po
 	$(MSGFMT) -o nls/it/$(PACKAGE).mo nls/it/$(PACKAGE).po
@@ -180,8 +180,8 @@
 	rm -f $(DESTDIR)$(PREFIX)/share/locale/es/LC_MESSAGES/$(PACKAGE).mo
 	rm -f $(DESTDIR)$(PREFIX)/share/locale/pt/LC_MESSAGES/$(PACKAGE).mo
 	rm -f $(DESTDIR)$(PREFIX)/share/locale/af/LC_MESSAGES/$(PACKAGE).mo
-	rm -f $(DESTDIR)$(PREFIX)/share/locale/nb_NO/LC_MESSAGES/$(PACKAGE).mo
-	rm -f $(DESTDIR)$(PREFIX)/share/locale/nn_NO/LC_MESSAGES/$(PACKAGE).mo
+	rm -f $(DESTDIR)$(PREFIX)/share/locale/nb/LC_MESSAGES/$(PACKAGE).mo
+	rm -f $(DESTDIR)$(PREFIX)/share/locale/nn/LC_MESSAGES/$(PACKAGE).mo
 	rm -f $(DESTDIR)$(PREFIX)/share/locale/it/LC_MESSAGES/$(PACKAGE).mo
 	rm -f $(DESTDIR)$(PREFIX)/share/locale/ru/LC_MESSAGES/$(PACKAGE).mo
 	rm -f $(DESTDIR)$(PREFIX)/share/locale/ro/LC_MESSAGES/$(PACKAGE).mo
@@ -192,8 +192,8 @@
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/es/LC_MESSAGES
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/pt/LC_MESSAGES
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/af/LC_MESSAGES
-	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/nb_NO/LC_MESSAGES
-	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/nn_NO/LC_MESSAGES
+	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/nb/LC_MESSAGES
+	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/nn/LC_MESSAGES
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/it/LC_MESSAGES
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/ru/LC_MESSAGES
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/ro/LC_MESSAGES
@@ -204,8 +204,8 @@
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/es
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/pt
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/af
-	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/nb_NO
-	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/nn_NO
+	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/nb
+	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/nn
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/it
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/ru
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(PREFIX)/share/locale/ro
@@ -259,8 +259,8 @@
 	mkdir -p $(DESTDIR)$(NLS_DIR)/es/LC_MESSAGES
 	mkdir -p $(DESTDIR)$(NLS_DIR)/pt/LC_MESSAGES
 	mkdir -p $(DESTDIR)$(NLS_DIR)/af/LC_MESSAGES
-	mkdir -p $(DESTDIR)$(NLS_DIR)/nb_NO/LC_MESSAGES
-	mkdir -p $(DESTDIR)$(NLS_DIR)/nn_NO/LC_MESSAGES
+	mkdir -p $(DESTDIR)$(NLS_DIR)/nb/LC_MESSAGES
+	mkdir -p $(DESTDIR)$(NLS_DIR)/nn/LC_MESSAGES
 	mkdir -p $(DESTDIR)$(NLS_DIR)/it/LC_MESSAGES
 	mkdir -p $(DESTDIR)$(NLS_DIR)/ru/LC_MESSAGES
 	mkdir -p $(DESTDIR)$(NLS_DIR)/ro/LC_MESSAGES
@@ -271,8 +271,8 @@
 	install -m 644 nls/es/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/es/LC_MESSAGES
 	install -m 644 nls/pt/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/pt/LC_MESSAGES
 	install -m 644 nls/af/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/af/LC_MESSAGES
-	install -m 644 nls/nb_NO/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/nb_NO/LC_MESSAGES
-	install -m 644 nls/nn_NO/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/nn_NO/LC_MESSAGES
+	install -m 644 nls/nb/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/nb/LC_MESSAGES
+	install -m 644 nls/nn/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/nn/LC_MESSAGES
 	install -m 644 nls/it/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/it/LC_MESSAGES
 	install -m 644 nls/ru/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/ru/LC_MESSAGES
 	install -m 644 nls/ro/$(PACKAGE).mo $(DESTDIR)$(NLS_DIR)/ro/LC_MESSAGES
diff -burN sysstat-5.0.3.orig/README sysstat-5.0.3/README
--- sysstat-5.0.3.orig/README	2004-04-09 22:37:10.966885272 +0200
+++ sysstat-5.0.3/README	2004-04-09 23:02:56.481931272 +0200
@@ -78,8 +78,8 @@
 ${PREFIX}/share/locale/es/LC_MESSAGES/sysstat.mo
 ${PREFIX}/share/locale/fr/LC_MESSAGES/sysstat.mo
 ${PREFIX}/share/locale/it/LC_MESSAGES/sysstat.mo
-${PREFIX}/share/locale/nb_NO/LC_MESSAGES/sysstat.mo
-${PREFIX}/share/locale/nn_NO/LC_MESSAGES/sysstat.mo
+${PREFIX}/share/locale/nb/LC_MESSAGES/sysstat.mo
+${PREFIX}/share/locale/nn/LC_MESSAGES/sysstat.mo
 ${PREFIX}/share/locale/pl/LC_MESSAGES/sysstat.mo
 ${PREFIX}/share/locale/pt/LC_MESSAGES/sysstat.mo
 ${PREFIX}/share/locale/ro/LC_MESSAGES/sysstat.mo
