--- sysstat-4.0.3/Makefile.orig	Sat Dec 29 17:48:18 2001
+++ sysstat-4.0.3/Makefile	Tue Jan 29 00:03:13 2002
@@ -24,17 +24,15 @@
 DESTDIR = $(RPM_BUILD_ROOT)
 BIN_DIR = $(PREFIX)/bin
 LIB_DIR = $(PREFIX)/lib
-ifndef MAN_DIR
-MAN_DIR = $(PREFIX)/man
-endif
+MAN_DIR = $(PREFIX)/share/man
 MAN1_DIR = $(MAN_DIR)/man1
 MAN8_DIR = $(MAN_DIR)/man8
 DOC_DIR = $(PREFIX)/doc/sysstat-$(VERSION)
 NLS_DIR = $(PREFIX)/share/locale
 
 # Compiler flags
-CFLAGS = -Wall -Wstrict-prototypes -pipe -O2 -fno-strength-reduce
-LFLAGS = -L. -lsysstat -s
+CFLAGS = -Wall -Wstrict-prototypes -pipe $(OPT_FLAGS) -fno-strength-reduce
+LFLAGS = -L. -lsysstat
 SAS_DFLAGS += -DSA_DIR=\"$(SA_DIR)\"
 
 # NLS (National Language Support)
@@ -43,11 +41,7 @@
 # The msgfmt command
 MSGFMT = msgfmt
 
-ifndef IGNORE_MAN_GROUP
-MANGRPARG = -g $(MAN_GROUP)
-else
 MANGRPARG =
-endif
 
 # Run-command directories
 ifndef RC_DIR
